include/util/static_rtree.hpp

```diff
@@ -631,6 +611,73 @@ class StaticRTree
     }
 
   private:
+    template <typename Callback>
```

Comment:
I just added `callback` parameter instead of returning vector and made this implementation private.